serverless.yml

```diff
@@ -17,6 +17,7 @@ provider:
     INCOMING_QUEUE_URL: ${env:INCOMING_QUEUE_URL}
     OUTGOING_QUEUE_URL: ${env:OUTGOING_QUEUE_URL}
     AWS_REGION: ${env:AWS_REGION}
+    IS_PROD: ${env:IS_SITE_LAUNCH_DEV}
```

Comment:
I was looking through the lambda functions to see if `IS_PROD` is being used anywhere and wasn't able to find it. Is this still necessary?

Reply:
opps: this was a typo, fixed [here]([4abf7bb](https://github.com/isomerpages/isomercms-backend/pull/538/commits/4abf7bbf969779ded606fdbacaff4e6fe62bb08a))